### ISSUE TITLE
Can't continue after authenticating with Ledger

### ISSUE THREAD

**Issue by Fang-** [urbit/bridge#140]
```
~satdef+ One of my friends is trying to get started, but can't auth with
         Bridge with his Ledger Nano S (1.5.5).  Are there any known issues
         with this?
~satdef+ After authing, 'continue' button never turns blue
...
~satdef+ palfun, he managed to auth when downgrading to 1.3.1
```

**Comment by Fang-** [comment]
Hmm, cannot reproduce with my own Ledger Nano S, which runs SE firmware 1.4.2 and MCU firmware 1.5. Continue button work just fine, regardless of the selected account. Contract/display data settings don't seem to matter.

I suspect ~satdef's friend was accessing Bridge over HTTP instead of HTTPS. Ledger refuses to authenticate in that case, and the continue button does stay greyed out. The Ledger login page gives instructions for this, and they're included in the repo readme as well.

**Comment by msutherl** [comment]
I ran through this flow recently and it was pretty fiddly. I can't remember exactly, but we ran into the above issue a few times until we got the settings and/or timing right.

**Comment by jtobin** [comment]
A Chrome update broke Ledger support awhile back -- I kind of figure it has been resolved since, but am not sure.  I last used a Ledger successfully w/Brave.

**Comment by shrugs** [comment]
_should_ be fixed as part of #155, but will wait for double-confirmation before closing this
